src/zar/main.go

```diff
@@ -0,0 +1,183 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"encoding/binary"
+	"encoding/gob"
+	"log"
+	"os"
+	"syscall"
+
+	// TODO: Change paths to be remotely imported from github
+	"manager"
+)
+
+// writeImage acts as the "main" method by creating and initializing the manager,
+// beginning the recursive walk of the directories, and writing the metadata header
+//
+// parameter (dir)	: the root dir name
+// parameter (output)	: the name of the image file
+// parameter (pageAlign): whether the files in the image will be page aligned
+// parameter (config)	: whether the image file is initialized from a config file
+// parameter (configPath): the path to the config file
+// parameter (format)	: the format of the config file
+func writeImage(dir string, output string, pageAlign bool, config bool, configPath string, format string) {
```

Comment:
We definitely need a global Config structure to avoid so many args. Not the priority now (but definitely need to do in the future.)

Reply:
Agreed.